src/ui/public/chrome/Tab.js

```diff
@@ -28,7 +29,16 @@ export default class Tab {
 
     this.lastUrlStoreKey = `lastUrl:${this.id}`;
     this.lastUrlStore = spec.lastUrlStore;
-    this.lastUrl = this.lastUrlStore ? this.lastUrlStore.getItem(this.lastUrlStoreKey) : null;
+
+    this.lastUrl = null;
+    if (this.lastUrlStore) {
+      this.lastUrl = this.lastUrlStore.getItem(this.lastUrlStoreKey);
+      if (this.lastUrl && !this.lastUrl.startsWith(this.rootUrl)) {
```

Comment:
this can get confused if the basePath changes, although arguably that could be intentional.  not worried about it, just making a note.